app/css/theme.css

```diff
@@ -25,3 +25,52 @@ a {
 .link-with-arrow::before {
   content: url('data:image/svg+xml; utf8, <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill="%23FF4470"><path d="M21.5 12.495L16 18h-3l4.498-4.5H3v-2h14.498L13 7h3z"/></svg>');
 }
+
+.hero,
+.main-nav {
+  background: #FFF;
+}
+
+.hero .hero-title,
+.link-with-arrow,
+.main-nav {
+  color: #FF4470;
+}
+
+.hero .hero-title::after {
+  background: #FF4470;
+}
+
+.grey-bg {
+  background: #f5f5f5;
+}
+
+h1.title {
+  font-size: 40px;
+  line-height: 48px;
+  max-width: 560px;
+  margin-bottom: 20px;
+  color: #FF4470;
+}
+h1.title:after {
+  content: "";
+  background: #222;
+  height: 1px;
+  width: 32px;
+  display: block;
+  margin-top: 16px;
+}
+h2.description {
+  margin-top: 20px;
+  margin-bottom: 40px;
+  max-width: 560px;
+  color: #999;
+}
+
+h1.title.center,
+h1.title.center:after,
+h2.center.description {
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
```

Comment:
In addition to all the comments from @keanulee, also...

Are the above styles (other than color and background) going to be shared across sites? If so they should be in `main.css` and the styles that are different across sites go to `theme.css`. For example,

main.css
```css
h1.title {
  font-size: 40px;
  line-height: 48px;
  max-width: 560px;
  margin-bottom: 20px;
}
```
theme.css
```css
h1.title {
  color: #FF4470;
}
```

Reply:
@frankiefu @keanulee thanks for your patience and guide me thru the process! 

To confirm, theme.css is only used for blog post & polymer-project website but not individual project site such as lit-html & litElement right?

Reply:
Those sites will be getting a theme.css added soon to be consistent will all the sites (e.g. https://github.com/Polymer/lit-html/pull/663/files#diff-f45ae56cba08cb08c9a79f3b20dfaa0c)

Reply:
Thanks, i've updated styling for h1.title in both css file according to Frankie's comment.

Reply:
👍